### DISCUSSION TITLE
Back cover?

### DISCUSSION THREAD

**Discussion by kyungnapark** [thegetty/quire#178] [Get Help] [answered]
Does quire have a back cover feature?

**Comment by geealbers** [comment, marked as answer]
Hi @pancaketents, unfortunately no, not at this time. At Getty, we've always had to do our cover wraps (front cover, spine and back cover for the print edition) in InDesign. For [a recent Quire book project I worked on for MCN](https://publications.mcn.edu/2020-scholars/), I did build a custom Quire template to output the cover wrap and it worked, but is very hacky and not in any shape to port to other projects. I'm hoping to get back to it, but will be while yet I'm afraid.

**Reply by kyungnapark** [reply]
I see, thank you!
